src/Plugin/Field/FieldFormatter/AddThisBasicToolboxFormatter.php

```diff
@@ -53,13 +53,12 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function viewElements(FieldItemListInterface $items) {
-    $settings = $this->getSettings();
-
-    $add_this_service = \Drupal::service('addthis.addthis');
-    $markup = $add_this_service->getBasicToolboxMarkup($settings);
 
     return array(
-      '#markup' => $markup
+      '#type' => 'addthis_basic_toolbox',
+      '#size' => $config['basic_toolbox']['buttons_size'],
```

Comment:
$config does not exist in the context of this function. Blocks use $this->configuration while Fields use $this->getSettings(). You will need to adjust these for the fields.